bindings/wasm/examples/worker.test.js

```diff
@@ -14,19 +14,11 @@
 
 import '@vitest/web-worker';
 
-import {WebIO} from '@gltf-transform/core';
 import {expect, suite, test} from 'vitest';
 
-import Module from './built/manifold.js';
-import {readMesh, setupIO} from './gltf-io';
 import {examples} from './public/examples.js';
 import ManifoldWorker from './worker?worker';
 
-const io = setupIO(new WebIO());
-
-const wasm = await Module();
```

Comment:
This is the weird part. For some reason if we load `manifold.js` multiple times, the subsequent loads will not complete.

Perhaps this is caused by pthread worker loader implementation. No idea about this, will try to come up with a minimal example and submit the issue to emscripten.

I currently workaround this issue by having only a single manifold worker, and move the get properties and genus calls to `worker.ts`, but this is not ideal.

Reply:
Did you ever submit this to emscripten?

Reply:
I forgot to try to reproduce this, will try later.